app/models/user.rb

```diff
@@ -18,7 +18,7 @@ class User < Common::RedisStore
   attribute :middle_name
   attribute :last_name
   attribute :gender
-  attribute :birth_date, Common::UTCTime
+  attribute :birth_date
```

Comment:
Common::UTCTime is an attribute type that I created. You should be able to just create your own coercer and have it format the input however you need.